build/packaging/rpm/gwms-factory.conf.httpd

```diff
@@ -18,6 +18,22 @@ Alias /factory /var/lib/gwms-factory/web-area
 </Directory>
 
 
+# TRACE and TRACK are not used in GlideinWMS and not recommended in production
+# Add the following line in httpd.conf to disable trace on the whole server
+# TraceEnable Off
+# This will disable unused methods using mod_rewrite
```

Comment:
Is this comment for the `TraceEnable` option or the `RewriteCondition`? Since it's in between the two, personally it seems a bit confusing as to which this comment is applicable

Reply:
Text improved to make more clear that refers to the following lines